.github/workflows/testcafe_tests.yml

```diff
@@ -62,6 +62,9 @@ jobs:
       fail-fast: false
       matrix:
         ARGS: [
+          { componentFolder: "accessibility", name: "accessibility" },
```

Comment:
why we adding new folder 'accessebility'? It's strange for me to have ci configuration like "componentFolder: accessebility" since it's not a component ⁠

Why not just navigation/treeView/accessebility as it was before?

Reply:
Because now we want to contain accessibility tests in one place. And see the reports in one place. That's a separate kind of test.

Maybe we should add a new folder with screenshot tests next to the accessibility folder. What do you think about that?